ModelBundle/Document/RouteDocument.php

```diff
@@ -0,0 +1,554 @@
+     */
+    public function getCondition()
+    {
+        return $this->condition;
+    }
+
+    /**
+     * @param string $condition
+     */
+    public function setCondition($condition)
+    {
+        $this->condition = $condition;
+    }
+
+    /**
+     * @return string
+     */
+    public function getHost()
+    {
+        return $this->host;
+    }
+
+    /**
+     * @param string $host
+     */
+    public function setHost($host)
+    {
+        $this->host = $host;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken0()
+    {
+        return $this->token0;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken1()
+    {
+        return $this->token1;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken2()
+    {
+        return $this->token2;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken3()
+    {
+        return $this->token3;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken4()
+    {
+        return $this->token4;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken5()
+    {
+        return $this->token5;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken6()
+    {
+        return $this->token6;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken7()
+    {
+        return $this->token7;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken8()
+    {
+        return $this->token8;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken9()
+    {
+        return $this->token9;
+    }
+
+    /**
+     * @return string
+     */
+    public function getToken10()
+    {
+        return $this->token10;
+    }
+
+    /**
+     * @return int
+     */
+    public function getWeight()
+    {
+        return $this->weight;
+    }
+
+    /**
+     * @return string
+     */
+    public function getNodeId()
+    {
+        return $this->nodeId;
+    }
+
+    /**
+     * @param string $nodeId
+     */
+    public function setNodeId($nodeId)
+    {
+        $this->nodeId = $nodeId;
+    }
+
+    /**
+     * @return string
+     */
+    public function getLanguage()
+    {
+        return $this->language;
+    }
+
+    /**
+     * @param string $language
+     */
+    public function setLanguage($language)
+    {
+        $this->language = $language;
+    }
+
+    /**
+     * @return string
+     */
+    public function getSiteId()
+    {
+        return $this->siteId;
+    }
+
+    /**
+     * @param string $siteId
+     */
+    public function setSiteId($siteId)
+    {
+        $this->siteId = $siteId;
+    }
+
+    /**
+     * @return string
+     */
+    public function getAliasId()
+    {
+        return $this->aliasId;
+    }
+
+    /**
+     * @param string $aliasId
+     */
+    public function setAliasId($aliasId)
+    {
+        $this->aliasId = $aliasId;
+    }
```

Comment:
The language prefix is a combination of the site and node configuration.

In this case, we only need to know the alias for right route name generation and so on.

The prefix is included in the pattern field